File: CPP/Petshop.cpp
#include <iostream>
#include <string>
using namespace std;

class PetShop {
    private:
        string id, nama_produk;
        double harga_produk;
        int stok_produk;
    
    public:
        // Konstruktor kosongan
        PetShop() : id(""), nama_produk(""), harga_produk(0.0), stok_produk(0) {}
    
        // Destruktor
        ~PetShop() {
    
        }
    
        void set_id(string id) { this->id = id; }
        void set_nama_produk(string nama) { this->nama_produk = nama; }
        void set_harga_produk(double harga) { this->harga_produk = harga; }
        void set_stok_produk(int stok) { this->stok_produk = stok; }
    
        string get_id() const { return id; }
        string get_nama_produk() const { return nama_produk; }
        double get_harga_produk() const { return harga_produk; }
        int get_stok_produk() const { return stok_produk; }
};

File: PHP/class/Petshop.php
<?php
class PetShop {
    private $id;
    private $nama_produk;
    private $harga_produk;
    private $stok_produk;
    private $foto_produk;

    public function __construct($id, $nama_produk, $harga_produk, $stok_produk, $foto_produk) {
        $this->id = $id;
        $this->nama_produk = $nama_produk;
        $this->harga_produk = $harga_produk;
        $this->stok_produk = $stok_produk;
        $this->foto_produk = $foto_produk;
    }

    public function get_id() {
        return $this->id;
    }

    public function get_nama_produk() {
        return $this->nama_produk;
    }

    public function get_harga_produk() {
        return $this->harga_produk;
    }

    public function get_stok_produk() {
        return $this->stok_produk;
    }

    public function get_foto_produk() {
        return $this->foto_produk;
    }
}
?>